### ISSUE TITLE
Ability to configure stage hazards behavior

### ISSUE THREAD

**Issue by Angstdiscord** [ThatNintendoNerd/stage_config#2]
Would be nice to enable or disable hazards on stages independent of your ruleset. 

An example would be forcing hazards on, despite hazards being turned off in the ruleset. 
or forcing hazards off even though your ruleset has them on. 

Should be configurable per stage.

**Comment by ThatNintendoNerd** [comment]
As of commit hash [5c4d9e4](https://github.com/ThatNintendoNerd/stage_config/commit/5c4d9e4a5faba3204e6482ae2209fbb86f54401b), this feature has been implemented into the codebase. A new release still needs to be published and a page on the wiki documenting the feature needs to be made, both before users can start implementing this feature into their stage mods.

Finalized configuration details are subject to change, but as of the aforementioned commit, the format is as such:
```toml
[gimmick_param]
Wario_Madein = { is_gimmick = false }
```

```toml
[gimmick_param]
WufuIsland = { is_gimmick = true }
```

I decided on a sub-table instead of a basic key-value pair like `is_flat_stage` in the event I decide to add new parameters to the feature, increasing future compatibility with configuration files.

This issue will be closed once the new release is published.
